<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="image">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>No file provided yet</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="fileButton">
            <property name="text">
             <string>Choose file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Predict with:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pSimpleButton">
            <property name="text">
             <string>Simple neural network</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pLogisticButton">
            <property name="text">
             <string>Logistic regression</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pSupportButton">
            <property name="text">
             <string>Support Vector Machine</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Transform:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sharpenButton">
            <property name="text">
             <string>Sharpen</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="gammaButton">
            <property name="text">
             <string>Adjust Gamma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="logButton">
            <property name="text">
             <string>Adjust Log</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sigmoidButton">
            <property name="text">
             <string>Adjust Sigmoid</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="intensButton">
            <property name="text">
             <string>Rescale Intensity</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eqhistButton">
            <property name="text">
             <string>Equalize Hist</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Not yet predicted</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
